payload/recipes/ibexa_install.bash

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+cd $PROJECTMAPPINGFOLDER
+
+PHP="php"
+COMPOSER="$PHP -d memory_limit=-1 /usr/local/bin/composer"
+VERSION=$1
+PACKAGE=$2
+PROVISIONING=$3
+DATABASE_PREFIXES=${DATABASE_PREFIXES:-DATABASE}
+
+for prefix in $DATABASE_PREFIXES
+do
+    DATABASE_NAME_VAR=${prefix}_NAME
+    DATABASE_HOST_VAR=${prefix}_HOST
+    DATABASE_USER_VAR=${prefix}_USER
+    DATABASE_PASSWORD_VAR=${prefix}_PASSWORD
+
+    # Wait for the DB
+    while ! mysqladmin ping -h"${!DATABASE_HOST_VAR}" -u"${!DATABASE_USER_VAR}" -p"${!DATABASE_PASSWORD_VAR}" --silent; do
+        echo -n "."
+        sleep 1
+    done
+    echo ""
+
+    mysql -h"${!DATABASE_HOST_VAR}" -u"${!DATABASE_USER_VAR}" -p"${!DATABASE_PASSWORD_VAR}" -e "CREATE DATABASE ${!DATABASE_NAME_VAR}"
+done
+
+echo "Installation eZ Platform ($PACKAGE - $VERSION) in the container"
+
+# Install ibexa/website-skeleton
+echo "Install ibexa/website-skeleton"
+$COMPOSER create-project --no-interaction ibexa/website-skeleton ezplatform $VERSION
+cd ezplatform
+# Copy nginx conf
+echo "Copy nginx conf"
+cp -r ../$PROVISIONING/dev/nginx/doc .
```

Comment:
I think we need to find a better solution somehow. We don't want to put the nginx config in Launchpad because it means we need to maintain the config that can change on Ibexa side.. that would be too complex to handle.

So we need to find a better way.

Reply:
apparently that is there: https://github.com/ibexa/docker/tree/main/templates/nginx/ez_params.d